### PR TITLE
RSDK-3378 - add docs converting custom remotes to modules

### PR DESCRIPTION
**Pull request by purplenicole730**
The Python docs currently has sections for both custom remotes and modules. To better support this, a new section explaining how to convert custom remotes to be a custom module component was added.
The steps necessary for this conversion are basically identical to some of the steps in the module component creation. These steps were written out in detail in that section, so it felt redundant to rewrite the steps completely. Instead, there is a basic example and a link to the pertaining sections.

### CODE REVIEW

File: docs/examples/example.ipynb [hunk 1]
```diff
@@ -905,6 +905,120 @@
...
...ch91OxQm36ohWNdOHGd7hj/l/SO+Or5lZ+jCpjdMlUVzRQt4miY67D1W10X3NGptKVdGErPQh3Lxv47sBHDR+QKA7tSkTp8oHIyaafSfVNZ82Ss6563regJVW1xnsVMrSsFNyhuQErDTQ1OahdrJ189bALl/o97xUrlE5YsDK+7ved1Qf0d+30OGz9z+Xj0Z9bGa3drrQqlOvjhmfN+cnGfrqG2bcreQSuu5H734sn0343MwODVhp6FNDmjrUcbr9uu72k5XytDtFd9BA0q//+9VMhuve0W2nr8m933XdlApY6b5SYwgX9In2s0GPOzXvmVdadZcVS1cYPq18lDlLpgCle89odZyBLw0y822Cqsm5Z7Y774vaJacO2m1Wyxcjd1mHu2EK+uEnYKWfLQ1uiPydQLu3LVOxTND+vBP79uwzn2taOcX02+osnPrxdBNC13ZvfTLMqY6Ww6zifibV1mqfTtUcHby/M97Aqds12q33N5LbH7rNtI30I6UCVlrBRSuYHggTPNZj0wDs5g2bnQpSuWWIE7oPDbi6xx/N/e6u4+c1Fu6xOnc/53E6rOsnYBXN75pr5f18efTZR2T00LGmAq/eo52cf7eEdu/srncqvk+42/bz6v57SLuKHjiun59Npdl1/Xz39/uZ7g1Y9Rv9WoKKle4fHiheuPCrn2P33qvRfn9OsxczhgdGwMoSk4CVJRTNEEAAAQQQQAABBBBAAAEEEEAAAZ8C6S1gldjpHnEql3z00UeyZ098VyEXXHCB1K5dO7FVrJcRsDoZsNq3d5+0uKel+QtwBdRutrS7rUgBqzGvj5WvP59prB9r9ahcHiHg1Nup+vDH70tMuy79Owe6anIfKGgXTSOcrsvCDRqaGPvGOLPIrbLjtvMGrDS0peGtaIZYBayi2Wcs2i7+7Q/p0yk+oFTr/FpOQCn+ga6fbUcbHlj55yrTtdv8ub+bLpMi7Tu0Gz5td/DAIenYtFOgWyANR2n3WZVrVpYKTpWPGudWT9AVkHf73sogOj9v/rymG5hKTnWkyjUqSbnK5YKCdN51ddwbUHuibVO55JpLTJM+nZwKbr8tNl32vfHh65I9R/jQk59z1x15A1bhKjqZg7H8kZyAlW56wvCJMvXDqWYv2h3iwPH9jVmkClZut4wZMmSUd6c6XW2GqRrmDR55g1Te4Frrl1s5IamE791
```
Comment (purplenicole730) [inline docs/examples/example.ipynb:1007]: um...

**Review by 8ashar — APPROVED** [review]

**Review by njooma — APPROVED** [review]
looks good, but do we want a few sentences on why someone might want to migrate away from remotes to modules? this just makes it seem like extra work but doesn't give benefits

**Review by dannenberg — APPROVED** [review]
this looks great! i think having the diff like that is very helpful. i agree with naveed that providing motivation would be nice. also maybe a couple comments around the new functions in the diff might be nice. is there a place i can view these docs rendered?

**Comment by purplenicole730** [comment]
> this looks great! i think having the diff like that is very helpful. i agree with naveed that providing motivation would be nice. also maybe a couple comments around the new functions in the diff might be nice. is there a place i can view these docs rendered?

@dannenberg If you go to the `Files Changed` tab and click the menu icon in the top right of whichever file you're looking at, there will be an option to "View File". That should give you a rendered view!

**Review by dannenberg — APPROVED** [review]
this looks fantastic. my only possible complaint (which may be a side effect of the github rendering vs out actual docs rendering) is: it's a little strange that ```from IPython.core.display import display, Image

display(Image(filename='./codediff.png'))``` shows up in the doc.

but again, this is wonderful. thank you for writing and helping me review it!

**Pull request merged**